java/core/src/main/java/com/google/protobuf/InlineMe.java

```diff
@@ -30,18 +30,30 @@
 
 package com.google.protobuf;
```

Comment:
Something went wrong with this file...I think I actually hit this same problem a few days ago and didn't have time to investigate, but it looks like the script somehow conflated these two files (ForceFieldBuildersPreRun.java and InlineMe.java).